sunpy/net/vso/attrs.py

```diff
@@ -353,11 +355,11 @@ def _(attr, results):
         if
         it.time.end is not None
         and
-        attr.min <= _attrs.Time.strptime(it.time.end, _TIMEFORMAT)
+        attr.min <= parse_time(it.time.end)
```

Comment:
Given we already know the format, I think using `Time.strptime` might be actually better here.

Reply:
I just pointed out in https://github.com/sunpy/sunpy/issues/3839#issuecomment-594110352 that the code was originally `astropy.time.Time`, but got accidentally changed to `_attrs.Time` in a bad edit, which was why it broke.

Reply:
yeah I agree, we don't need to use `parse_time` we just need to undo the change as @ayshih pointed out in the issue.

Reply:
@vn-ki Check my comment in https://github.com/sunpy/sunpy/issues/3839.